docs/expressions.md

```diff
@@ -330,6 +345,20 @@ Remove any values lower than or equal to number from a series. Will error if thi
 
 Remove any NaN or Inf values from a series. Will error if this operation results in an empty series.
 
+## epoch()
+
+Returns the Unix epoch in seconds of the expression start time (scalar).
+
+## filter(series, number)
+
+Returns all results in series that are a subset of anything in number, or
+that have number as a subset. Useful with the limit and sort functions to
+return the top X results of a query.
+
+## limit(number, count)
```

Comment:
Read: http://bosun.org/expressions.html#data-types

On Tue, Jun 2, 2015 at 11:30 AM, Greg Bray <notifications@github.com> wrote:

> In docs/expressions.md
> <https://github.com/bosun-monitor/bosun/pull/1023#discussion_r31535459>:
>
> > @@ -330,6 +345,20 @@ Remove any values lower than or equal to number from a series. Will error if thi
> >
> >  Remove any NaN or Inf values from a series. Will error if this operation results in an empty series.
> >
> > +## epoch()
> > +
> > +Returns the Unix epoch in seconds of the expression start time (scalar).
> > +
> > +## filter(series, number)
> > +
> > +Returns all results in series that are a subset of anything in number, or
> > +that have number as a subset. Useful with the limit and sort functions to
> > +return the top X results of a query.
> > +
> > +## limit(number, count)
>
> Maybe we need to define what number is... because I'm not sure I've ever
> seen a definition for number vs series
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/bosun-monitor/bosun/pull/1023/files#r31535459>.
>